source/options.html

```diff
@@ -15,6 +15,12 @@
 			Show only participating notifications
 		</label>
 	</p>
+	<p>
+		<label>
+			<input type="checkbox" name="closeOnMouseleave">
+			The popup to close when you move the mouse away from its area
```

Comment:
This sounds strange to me. Maybe write it like this?

```suggestion
			Automatically close the popup on mouse leave
```

Reply:
I agree my wording was not a very good choice.

But, doesn't "on mouse leave" sound like a technical/programming term? IMHO it's not self-explanatory enough. 
Maybe change it to the following instead?

> Automatically close the popup when mouse is moved away

What do you think?

Reply:
I used my new wording.